.github/workflows/build.yml

```
@@ -30,57 +30,20 @@ jobs:
     needs: [ BuildSdk ]
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v4
+      - id: set-matrix
+        run: python3 Buildscripts/gh-generate-device-matrix.py
+  BuildFirmware:
+    runs-on: ubuntu-latest
+    needs: GenerateDeviceMatrix
+    strategy:
+      matrix: ${{ fromJSON(needs.generate-device-matrix.outputs.matrix) }}
```

actionlint (1.7.12)

[error] property "generate-device-matrix" is not defined in object type {generatedevicematrix: {outputs: {matrix: string}; result: string}}

(expression)